src/apps/backend/modules/comment/comment_service.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
+
+from modules.comment.errors import CommentCreationError, CommentNotFoundError, CommentServiceError, CommentUpdateError
+from modules.comment.internal.comment_reader import CommentReader
+from modules.comment.internal.comment_writer import CommentWriter
+from modules.comment.types import Comment, CreateCommentParams
+
+
+class CommentService:
+    @staticmethod
+    def create_comment(*, params: CreateCommentParams) -> Comment:
+        try:
+            comment = CommentWriter.create_comment(params=params)
+            return comment
```

Comment:
Nit: No need to store `comment` in variable, you should be able to simply write:

```
return CommentWriter.create_comment(params=params)
```